testing/MLDB-1359_procedure_latest_run.py

```diff
@@ -0,0 +1,59 @@
+#
+# MLDB-1359_procedure_latest_run.py
+# Mich, 2016-02-05
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+import time
+from dateutil import parser as date_parser
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+
+class ProcedureLatetRunTest(MldbUnitTest): # noqa
+
+    def test_it(self):
```

Comment:
I thought we were going away from monolithic tests.  Consider splitting this tests into two or three?